biz.aQute.bndlib/src/aQute/bnd/osgi/AnnotationHeaders.java

```diff
@@ -58,6 +61,8 @@
  */
 class AnnotationHeaders extends ClassDataCollector implements Closeable {
 
+	static final Pattern SIMPLE_PARAM_PATTERN = Pattern.compile("\\$\\{(\\w*)\\}");
```

Comment:
Shouldn't this be: `\\$\\{(\\p{javaJavaIdentifierStart}\\p{javaJavaIdentifierPart}*)\\}` since it (1) requires at least one character and (2) has an alphabet which matches the alphabet of the names of annotation elements?